volmdlr/wires.py

```diff
@@ -3764,6 +3866,29 @@ def merge_with(self, contour3d):
 
         return contours
 
+    def primitive_over_contour(self, primitive):
+        '''
+        copied from Contour2D
```

Comment:
Why not factor in common class?

Reply:
@masfaraud May-be, I can put it in Contour ?